testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/AbstractKeycloakTest.java

```diff
@@ -285,7 +285,7 @@ public void deleteAllCookiesForMasterRealm() {
     protected void deleteAllCookiesForRealm(String realmName) {
         // we can't use /auth/realms/{realmName} because some browsers (e.g. Chrome) apparently don't send cookies
         // to JSON pages and therefore can't delete realms cookies there; a non existing page will do just fine
-        navigateToUri(oauth.SERVER_ROOT + "/auth/realms/" + realmName + "/super-random-page");
+        navigateToUri(oauth.SERVER_ROOT + "/auth/realms/" + realmName + "/protocol/openid-connect/3p-cookies/step1.html");
```

Comment:
You likely don't want to use this URL as this iframe sets cookies and does redirects.

Reply:
What cookies are set there?

Reply:
It sets the `KEYCLOAK_3P_COOKIE_SAMESITE` and `KEYCLOAK_3P_COOKIE` (see [source code](https://github.com/keycloak/keycloak/blob/904e688400cbb4470b15d88ef9f65f03ee1e7c71/services/src/main/resources/org/keycloak/protocol/oidc/endpoints/3p-cookies-step1.html#L49-L56)), and then redirects to `step2.html`, which then attempts to [read and clear](https://github.com/keycloak/keycloak/blob/904e688400cbb4470b15d88ef9f65f03ee1e7c71/services/src/main/resources/org/keycloak/protocol/oidc/endpoints/3p-cookies-step2.html#L12-L15) those cookies.

> The main idea here is to actually reach out the server and a realm to make sure the driver can delete all cookies bound to a domain/path. 

Aren't we just testing WebDriver at that point? Seems rather redundant.

Reply:
Not sure if redundant because that method is called before, and sometimes after, a test is run. So it seems all about making sure the webdriver is not referencing cookies from previous runs. And by cookies, the main ones are those related to the user session. But again, that is what I would like to check with Marek.

I can choose some other HTML-facing endpoint if we want to. But those cookies you mentioned should not interfere with tests as they are expiring.

I was able to run the test I'm fixing in this PR multiple times without any failure. Without the change, it fails very often. We can always evaluate a better approach if this one proves to be not enough.

Reply:
@jonkoops I tried to change the endpoint locally to `step2.html` and it works fine too. Not sure if makes any difference to change the PR but if you have a suggestion to use some other endpoint, I can see how it goes.

Reply:
@jonkoops Is it ok with you to use `step2.html` url or will it also create cookies like `KEYCLOAK_3P_COOKIE_SAMESITE` ?

Other option is to create some "dummy" endpoint (maybe in `TestingResourceProvider` or somewhere where our other "test endpoints" are), but would be nice to avoid that, so we don't create endpoint just for this purpose?

Reply:
Yeah using `step2.html` is an option, but in general I would not recommend testing against these endpoints. If it is ever changed this test will fall over.

In general I think we 'abuse' random endpoints to much. For example, I have to refactor a bunch of unrelated tests every-time I change a front-end, as other tests do stupid things like use it as a crutch to make assertions on page contents that are likely to change.

If there is a page that can be used from `TestingResourceProvider` that seems more appropriate.